The wrong variable is saved before the test and CFLAGS gets overwritten
with an empty one afterwards.

--- configure.in.orig	2020-05-11 15:34:13.000000000 +0200
+++ configure.in	2020-05-21 08:45:20.113154658 +0200
@@ -604,7 +604,7 @@
    fi
 fi
 
-SAVE_CPPFLAGS="$CPPFLAGS"
+save_CFLAGS="$CFLAGS"
 CFLAGS=`python-config --includes`
 if [ $? -eq 0 ] 
 then
